frontend/src/main/scala/bloop/data/Project.scala

```diff
@@ -186,7 +186,7 @@ final case class Project(
 
   def jdkConfig: Option[JdkConfig] = {
     platform match {
-      case Platform.Jvm(config, _, _, _, _) => Some(config)
+      case Platform.Jvm(config, _, _, _, _, _) => Some(config)
```

Comment:
nit: would it make sense to refactor into `case jvm: Platform.Jvm => Some(jvm.config)`?